alimasa_eliza/countries.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Countries</title>
+    <link href="countries_style.css" rel="stylesheet" />
+</head>
+<body>
+    <div class="countries">
+        <h1>Countries of the World</h1>
+        <div>
+            <input type="text" id="country_input" 
+            placeholder="Please enter a country">
+            <button onclick="countrySearch()">Search</button>
+        </div>
```

Comment:
Add a few indent spaces if the element has multiple lines. 
```suggestion
        <div>
            <input type="text" id="country_input" 
                   placeholder="Please enter a country">
            <button onclick="countrySearch()">Search</button>
        </div>
```